go.mod

```diff
@@ -6,6 +6,7 @@ go 1.18
 replace (
 	github.com/Microsoft/go-winio => github.com/endocrimes/go-winio v0.4.13-0.20190628114223-fb47a8b41948
 	github.com/apparentlymart/go-textseg/v12 => github.com/apparentlymart/go-textseg/v12 v12.0.0
+	github.com/hashicorp/consul/api => github.com/fredwangwang/consul/api v1.12.1
```

Comment:
Is this `replace` a leftover of some local testing?

Reply:
see https://github.com/hashicorp/consul/issues/13305
consul api package not yet expose the feature to configure customized tls, tho the feature is available staring consul 1.11.2+. Need some "push" to get the new api package published and i can remove this line 😃